bundle/bin/rke2-uninstall.ps1

```diff
@@ -345,6 +365,17 @@ function Remove-Namespace() {
     Invoke-Ctr -cmd "namespace remove $namespace"
 }
 
+function Create-Lockfile() {
+    $fullPath = Join-Path -Path "C:\var\lib\rancher\rke2\agent\" -ChildPath "kubelet.kubeconfig"
+    if (-not ($fullPath)) {
+        # For cases where Datadir is not the default location (can take a minute to find)
+        $fullPath = Get-ChildItem -Path C:\ -Recurse -ErrorAction SilentlyContinue | Where-Object { $_.Name -eq $file } | Select-Object -ExpandProperty FullName
```

Comment:
Doing a full disk search on the C: is frightening, lets not do this. We can't even guarantee that it's on this disk, if they change the data-dir they are VERY likely to put it on a different drive letter.

Can we perhaps look at the kubelet process to find its executable path, and determine the data-dir from that?

Reply:
good idea, actually we are checking a bit before if `ctr` command exists, we can fetch dataDir from there and place the lockFile in that directory